src/client/components/header/search/Search.js

```diff
@@ -66,4 +85,19 @@ class Search extends React.Component {
   }
 }
 
+let mapStateToProps = (state) => {
```

Comment:
please use `const` instead of let. The diff between const is that you can change it.